README.md

```diff
@@ -29,84 +29,62 @@ Instructions on setting up the project and getting it running on a local machine
 - Set the `COUCH_URL` environment variable to point your test instance, for example it something similar to this: `http://[user]:[pass]@[host]:[port]/medic`
 - Double-check your CHT test instance is running
 - Install the project by running `npm ci`
-- Design the test data in `custom-design.ts`. See section [Designing Test Data](#designing-test-data).
-- Build, generate data and upload by running `npm run generate`
+- Design the test data in a custom JavaScript file. See section [Designing Test Data](#designing-test-data).
+- Build, generate data and upload by running `npm run generate *path_to_your_custom_design_file*`
 
 ## Designing Test Data
 
 Steps to design the test data:
 
-1. Open `src/custom-design.ts` - not the ideal way but to be improved in the future.
-2. Create and export a class that extends the `DocDesign` abstract class. 
-```ts
-export class CustomDesign extends DocDesign {
-  getDesign(): DesignSpec[] {
-    return [];
-  }
-}
-```
-3. Update the `getDesign` function to return an array that adhere with the [DesignSpec interface](./src/doc-design.ts).
+1. Create a custom JavaScript file that exports a `default` function. This function should match the [`DocDesign` type](./src/doc-design.ts).
+2. Your custom function should return an array of [`DesignSpec`](./src/doc-design.ts) objects that define the structure of your data to create.
+3. Generate and upload data `npm run generate *path_to_your_custom_design_file*`. That's all! You can check the data in the CHT's CouchDB.
 
-| Property | Required | Explanation |
```

Comment:
Happy to add this table back in here if you think it is helpful, but I ended up expanding the documentation for the various types defined in `doc-design.ts` (which I linked to above). It seemed unnecessary to have the documentation in two places (and here in the README seems more likely to get stale over time...)

Reply:
Your suggestions are on point for the target audience - experienced application builders or CHT developer contributors. Thank you!